index.html

```diff
@@ -221,9 +225,85 @@ <h1>Skills</h1>
                 <li>gitlab</li>
                 <li>Terminal</li>
               </ul>
-            </div>
           </section>
-          </div>
-          </body>
-          </html>
-         
\ No newline at end of file
+          <section class="touch">
+            <h2>
+              I'm always interested in hearing about new projects, so if you'd like
+              to chat please get in touch.
+            </h2>
+            <form action="https://formspree.io/f/{form_id}" method="post">
+              <ul class="touchForm">
+                <li>
+                  <input
+                    type="text"
+                    name="name"
+                    class="input-Text"
+                    placeholder="Full name"
+                    required
+                    maxlength="30"
+                    type="text"
+                  />
+                </li>
+                <li>
+                  <input
+                    class="input-Text"
+                    type="email"
+                    name="email"
+                    placeholder="Email address"
+                    required
+                    type="email"
+                  />
+                </li>
+                <li>
+                  <textarea
+                    class="inputTextarea"
+                    name="opinion"
+                    id="opinion"
+                    cols="50"
+                    rows="10"
+                    placeholder="Enter text here"
+                    maxlength="500"
+                    required
+                    type="text"
+                  ></textarea>
+                </li>
+                <li>
+                  <button type="button" class="touch-button">Get In Touch</button>
```

Comment:
- [ ] I kindly suggest that you add the correct button interactions including hover and active (when it's pressed), as shown in the Figma template below:

<img width="541" alt="image" src="https://user-images.githubusercontent.com/52934128/193900851-3cb52cfc-3deb-4bfa-9ad8-054f39c6ea76.png">

the button on the pressed state has the `background: #E05304` and on hover has the text `color: #F55800`

### EXAMPLE:
**for pressed state**
```
button:active {
   background-color: #E05304;
}
```
**for hover state**
```
button:hover {
    color: #F55800;
}
```